ibm/service/project/resource_ibm_project.go

```diff
@@ -0,0 +1,556 @@
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	getProjectOptions := &projectv1.GetProjectOptions{}
+
+	getProjectOptions.SetID(d.Id())
+
+	getProjectResponse, response, err := projectClient.GetProjectWithContext(context, getProjectOptions)
+	if err != nil {
+		if response != nil && response.StatusCode == 404 {
+			d.SetId("")
+			return nil
+		}
+		log.Printf("[DEBUG] GetProjectWithContext failed %s\n%s", err, response)
+		return diag.FromErr(fmt.Errorf("GetProjectWithContext failed %s\n%s", err, response))
+	}
+
+	if err = d.Set("name", getProjectResponse.Name); err != nil {
+		return diag.FromErr(fmt.Errorf("Error setting name: %s", err))
+	}
+	if !core.IsNil(getProjectResponse.Description) {
+		if err = d.Set("description", getProjectResponse.Description); err != nil {
+			return diag.FromErr(fmt.Errorf("Error setting description: %s", err))
+		}
+	}
+	if !core.IsNil(getProjectResponse.Configs) {
+		configs := []map[string]interface{}{}
+		/*
+			for _, configsItem := range getProjectResponse.Configs {
+				configsItemMap, err := resourceIbmProjectProjectConfigInputToMap(&configsItem)
+				if err != nil {
+					return diag.FromErr(err)
+				}
+				configs = append(configs, configsItemMap)
+			}
+		*/
+		if err = d.Set("configs", configs); err != nil {
+			return diag.FromErr(fmt.Errorf("Error setting configs: %s", err))
+		}
+	}
+	if !core.IsNil(getProjectResponse.Crn) {
+		if err = d.Set("crn", getProjectResponse.Crn); err != nil {
+			return diag.FromErr(fmt.Errorf("Error setting crn: %s", err))
+		}
+	}
+	if !core.IsNil(getProjectResponse.Metadata) {
+		metadataMap, err := resourceIbmProjectProjectMetadataToMap(getProjectResponse.Metadata)
+		if err != nil {
+			return diag.FromErr(err)
+		}
+		if err = d.Set("metadata", []map[string]interface{}{metadataMap}); err != nil {
+			return diag.FromErr(fmt.Errorf("Error setting metadata: %s", err))
+		}
+	}
+
+	return nil
+}
+
+func resourceIbmProjectUpdate(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	projectClient, err := meta.(conns.ClientSession).ProjectV1()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	updateProjectOptions := &projectv1.UpdateProjectOptions{}
+
+	updateProjectOptions.SetID(d.Id())
+
+	hasChange := false
+
+	if hasChange {
+		_, response, err := projectClient.UpdateProjectWithContext(context, updateProjectOptions)
+		if err != nil {
+			log.Printf("[DEBUG] UpdateProjectWithContext failed %s\n%s", err, response)
+			return diag.FromErr(fmt.Errorf("UpdateProjectWithContext failed %s\n%s", err, response))
+		}
+	}
+
+	return resourceIbmProjectRead(context, d, meta)
+}
+
+func resourceIbmProjectDelete(context context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	projectClient, err := meta.(conns.ClientSession).ProjectV1()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	deleteProjectOptions := &projectv1.DeleteProjectOptions{}
+
+	deleteProjectOptions.SetID(d.Id())
+
```

Comment:
We would encourage you to provide `destroy` flag feature like in api.
Is there any specific reason for not including it?